src/components/collapse/package.json

```diff
@@ -10,27 +10,43 @@
         "events": [
             {
                 "event": "show",
-                "description": "emitted when collaspe has started to open"
+                "description": "Emitted when collapse has started to open"
             },
             {
                 "event": "shown",
-                "description": "emitted when collaspe has finished opening"
+                "description": "Emitted when collapse has finished opening"
             },
             {
                 "event": "hide",
-                "description": "emitted when collaspe has started to close"
+                "description": "Emitted when collapse has started to close"
             },
             {
                 "event": "hidden",
-                "description": "emitted when collaspe has finished closing"
+                "description": "Emitted when collapse has finished closing"
             },
+            {
+                "event": "bv::collapse::state",
+                "description": "Emitted when collapse has changed its state",
```

Comment:
This description should note that the event is emitted on `$root` not the component instance.